rfcs/0000-rfc-template.md

```diff
@@ -22,6 +23,37 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 -->
 
 ## Fields
+The `vulnerabilities` fields being proposed are as follows:
+
+Field | Type | Description /Use Case
+vulnerability.vendor.id | keyword | A vulnerability doesn't have necessary a CVE associated with it. I don't know if it makes sense to seperate vulnerability ID (like CVEs) to the vendor/detection IDs.
+vulnerability.title	| keyword | Title/Name/Short Description for vulnerability, to be used in flyout and dashboards.
+vulnerability.mitigation | text | Explains user how to fix or mitigate the problem, could be usefd to store resolution from the scanner vendor or document mitigation in place
+vulnerability.published	| date | The “published” field indicates the date when information about a specific vulnerability was publicly disclosed or made available.It represents the moment when details about the vulnerability were shared with the security community, vendors, and the public.This field helps security professionals track the timeline of vulnerability awareness.Example value: If a critical vulnerability (e.g., CVE-2024-1234) affecting a widely used software is disclosed on February 1, 2024, the published date for that CVE would be February 1, 2024.
+vulnerability.patch.* | object
+vulnerability.patch.exists | boolean | The “patch” field refers to whether a security fix or update (commonly known as a patch) is available to address the identified vulnerability. It indicates whether the software vendor or developer has released a solution to mitigate the vulnerability.
+vulnerability.patch.name | text | Name of the patch
+vulnerability.patch.code | keyword | Associated patch code for example ESA-2020-13
+vulnerability.evidence	| text | A demonstration of the validity of a vulnerability claim, e.g. app.any.run replaying the exploitation of the vulnerability.
+vulnerability.status | keyword | The status field helps security teams track vulnerabilities, prioritize actions, and communicate their progress effectively
+vulnerability.tags	| keyword | This is different from cloud provider assigned resource tags, this is specifically for vulnerability. Vulnerability tags serve as a way to add custom metadata to vulnerabilities, enhancing their context and aiding in search and automation. 
```

Comment:
What would be the added value of this field compared to the default tags one? We can create specific combinaison like `vulnerability:exploited`. The only added value I see is for dashboarding as it makes things much easier.